### PR TITLE
llvm 3.6.0

### PR DESCRIPTION
**Pull request by lembacon**

### CODE REVIEW

**Comment by petere** [comment]
works for me

File: llvm36.rb [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+  option :universal
+  option "with-lld", "Build LLD linker"
+  option "with-lldb", "Build LLDB debugger"
+  option "with-asan", "Include support for -faddress-sanitizer (from compiler-rt)"
+  option "with-all-targets", "Build all target backends"
+  option "with-python", "Build lldb bindings against the python in PATH instead of system Python"
+  option "without-shared", "Don't build LLVM as a shared library"
+  option "without-assertions", "Speeds up LLVM, but provides less debug information"
+
+  deprecated_option "all-targets" => "with-all-targets"
```
Comment (DomT4) [inline llvm36.rb]: You can cut these three lines. Since it's technically a new formula here, nothing has been deprecated yet.

Comment (lembacon) [inline llvm36.rb]: removed via ee3d6c5 :smile:

Comment (DomT4) [inline llvm36.rb]: Thanks! Once the test bot has given it the green tick again, I'll merge this in.

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, We appreciate it! For future reference the preferred commit message for new formulae is to tack the words ` (new formula) ` onto the end of ` llvm 3.6.0 `, but don't worry about it this time around :smiley_cat:

**Pull request closed without merging**